.github/workflows/ci.yml

```diff
@@ -25,32 +24,36 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
```

Comment:
Push back on this if you want. 12 is EOL, not sure there's much sense in supporting it anymore (but I could be convinced otherwise).

Reply:
We should be doing 14, 16, 18

Reply:
This comment is dated, I added 18 to this matrix after leaving the comment 👍